packages/server/src/fhir/repo.ts

```diff
@@ -2455,7 +2459,20 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
         // Attempt 2: 50 * (2^1) = 100 * [0.75, 1] = **[75, 100] ms**
         // etc...
         const delayMs = Math.ceil(baseDelayMs * 2 ** attempt * (0.75 + Math.random() * 0.25));
+        getLogger().info('Retrying transaction after delay', {
+          attempt,
+          attemptDurationMs,
+          transactionAttempts,
+          delayMs,
+          baseDelayMs,
+        });
         await sleep(delayMs);
+      } else {
+        getLogger().info('Transaction failed after final <attempt></attempt>', {
```

Comment:
What's the HTML-esque tag doing?

Reply:
That was an errant VSCode extension that wanted to auto-expand "attempt" to a tag for some reason. 🤷